src/Backend/Modules/ContentBlocks/Actions/ContentBlockAdd.php

```diff
@@ -43,27 +38,55 @@ public function execute(): void
         }
 
         /** @var CreateContentBlock $createContentBlock */
```

Comment:
I don't think we need this anymore since the method has a return type and that gives us the autocompletion that this provided in the past